src/joint_infer.jl

```diff
@@ -595,12 +660,14 @@ function process_sources_dynamic!(images::Vector{Model.Image},
                 end
             end
             Log.info("Batch $(batch) - $(process_sources_elapsed_times)")
-            idle_percent = 100.0 * (maximum(process_sources_elapsed_times) -
-                                      mean(process_sources_elapsed_times)) /
-                                      maximum(process_sources_elapsed_times)
-            Log.info("Batch $(batch) avg threads idle: $(round(Int, idle_percent))%")
+	    avg_thread_idle_time = mean(maximum(process_sources_elapsed_times) - process_sources_elapsed_times)
+	    maximum_thread_time = maximum(process_sources_elapsed_times)
+            idle_percent = 100.0 * (avg_thread_idle_time / maximum_thread_time)
+            Log.info("Batch $(batch) avg threads idle: $(round(Int, idle_percent))% ($(avg_thread_idle_time) / $(maximum_thread_time))")
+	    total_idle_time += sum(maximum(process_sources_elapsed_times) - process_sources_elapsed_times)
         end
     end
+    Log.info("Total idle time: $(total_idle_time)")
```

Comment:
Would you print total idle time as a proportion of total time? Maybe better to round to the nearest second---makes it easier to see what's important.